client/src/components/style/App.css

```diff
@@ -1,3 +1,36 @@
+
+/*********** modal max size ***********/
+.modal {
+    height: 625px;
+    width: 1200px;
+}
+
+/*********** modal image ***********/
+
+.overlay {
+    grid-column:  1 / span 1;
+    max-width: 900px; 
+    height: auto; 
+}
+
+.overlay:hover {
+    cursor: pointer 
+}
+
+/* .image {
```

Comment:
Is this needed for later?

Reply:
Yes, I left in inline to test something and forgot to change it back.